Assets/Scripts/playerController.cs

```diff
@@ -7,51 +7,63 @@ public class playerController : MonoBehaviour
      public Animator Animator;
     [SerializeField]
     float Jump_Power;
+    [SerializeField]
+    float Speed;
+    Rigidbody2D RB2d;
 
-    void PlayingAnimation() {
-        float speed = Input.GetAxisRaw("Horizontal");
-        float jump = Input.GetAxisRaw("Vertical");
-        Animator.SetFloat("Speed", Mathf.Abs(speed));
+    private void Start()
+    {
+        RB2d = gameObject.GetComponent<Rigidbody2D>();
+    }
+    void PlayerMovement(float Horizonatal)
+    {
+        Vector2 pos = transform.position;
+        pos.x += Horizonatal * Speed * Time.deltaTime;
+        transform.position = pos;
+    }
+    void PlayingAnimation(float Horizonatl) {
```

Comment:
Write access modifiers when writing the method.

Reply:
yeah sometimes I don't do that, sorry for that